calendra/europe/denmark.py

```diff
@@ -26,5 +26,6 @@ def get_store_bededag(self, year):  # 'great prayer day'
 
     def get_variable_days(self, year):
         days = super().get_variable_days(year)
-        days.append((self.get_store_bededag(year), "Store Bededag"))
+        # Great prayer day is a non-working day since 2024
```

Comment:
Do you mean it's a working day since 2024?

```suggestion
        # Great prayer day is a working day since 2024
```